lib/nacre.rb

```diff
@@ -3,9 +3,12 @@
 require 'nacre/configuration'
 require 'nacre/connection'
 
+require 'nacre/journal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.